datadog_checks_base/tests/conftest.py

```diff
@@ -11,3 +12,36 @@
 def socks5_proxy():
     with docker_run(compose_file=os.path.join(HERE, "compose", "socks5-proxy.yaml")):
         yield "proxy_user:proxy_password@localhost:1080"
+
+
+@pytest.fixture(scope="session")
+def kerberos():
+
+    compose_file = os.path.join(HERE, "compose", "kerberos.yaml")
+    realm = "EXAMPLE.COM"
+    svc = "HTTP"
+    webserver_hostname = "web.example.com"
+    krb5_conf = os.path.join(HERE, "krb5.conf")
+
+    common_config = {
+        "url": "http://localhost:80",
+        "keytab": os.path.join("/tmp", "shared-volume", "http.keytab"),
+        "cache": os.path.join("/tmp", "shared-volume", "krb5cc_web"),
```

Comment:
Let's use our utility for temporary directories like https://github.com/DataDog/integrations-core/blob/31ad373f298128f2924d3f36541d2e8452678cc9/openldap/tests/conftest.py#L18